libvast/native-plugins/summarize.cpp

```diff
@@ -750,10 +796,22 @@ class plugin final : public virtual pipeline_operator_plugin {
 
   [[nodiscard]] caf::expected<std::unique_ptr<pipeline_operator>>
   make_pipeline_operator(const record& config) const override {
-    auto parsed_config = configuration::make(config);
-    if (!parsed_config)
-      return parsed_config.error();
-    return std::make_unique<summarize_operator>(std::move(*parsed_config));
+    std::vector<std::string> sections_to_reformat;
+    for (const auto& key : {"min", "max", "any", "all", "sum"}) {
+      if (config.contains(key))
+        sections_to_reformat.emplace_back(key);
+    }
+    // new format detected. Proceed as usual
+    if (sections_to_reformat.empty()) {
+      return make_summarize_operator(config);
+    }
```

Comment:
Style nit: no braces for single-line scopes.

Reply:
I wonder if i will ever get used to this :D